lapce-proxy/src/plugin/wasi.rs

```diff
@@ -1,3 +1,6 @@
+#[cfg(test)]
+mod tests;
```

Comment:
This shouldn't be here?

Reply:
At the beginning of the file? I think it's ok. We won’t find it at the very end, because I put the tests in a separate file.

Reply:
Here is an example of how a module starts with a test submodule declaration (https://github.com/rust-lang/rust/blob/master/library/std/src/collections/hash/map.rs#L2)